include/tvm/meta_schedule/task_scheduler.h

```diff
@@ -96,6 +98,7 @@ class TaskSchedulerNode : public runtime::Object {
     v->Visit("cost_model", &cost_model);
     v->Visit("measure_callbacks", &measure_callbacks);
     v->Visit("num_trials_already", &num_trials_already);
+    // v->Visit("logging_func", &logging_func);
```

Comment:
explicitly say it's skipped 

```suggestion
    // `logging_func` is not visited
```